cranelift/filetests/filetests/runtests/issue5523.clif

```diff
@@ -0,0 +1,11 @@
+test interpret
+test run
+target riscv64
```

Comment:
Just for good coverage while we're here, can we add the other targets (`x86_64`, `aarch64`, `s390x`) here too?

Reply:
Ok